superset-frontend/src/explore/components/controls/VizTypeControl/constants.tsx

```diff
@@ -33,7 +33,7 @@ export const FEATURED_CHARTS: VizMeta[] = [
   { name: VizType.Table, icon: <Icons.TableOutlined iconSize="l" /> },
   {
     name: VizType.BigNumberTotal,
-    icon: <Icons.BigNumberChartTile />,
+    icon: <Icons.BigNumberChartTile iconSize="l" viewBox="0 0 16 14" />,
```

Comment:
NOTE: not sure how this works exactly, but assuming this icon ends up in an `Button.icon`, I think antd's internal might adjust the icon size it receives based on the button size props. Just pointing out it could be possibly overridden there (?) maybe only if it's not already set (?) Not sure how it all works but curious to clarify or check it works as expected.